src/Red.jsx

```diff
@@ -0,0 +1,37 @@
+import { React, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+
+export default function Red(props) {
+  const { AllshoesName, AllImage } = props;
+
+  return (
+    <>
+      <div className="container">
+        <div className="listContainer">
+          {AllshoesName.map((ele) => {
+            if (ele.color === "RED") {
+              return (
+                <div className="infoCard">
```

Comment:
Because `.map()` returns an array of elements, each element needs to have a unique `key` or else React can behave strangely.

```suggestion
                <div className="infoCard" key={ele.id}>
```